server/controllers/errorsController.js

```diff
@@ -97,40 +92,42 @@ errorsController.saveErrors = async (req, res, next) => {
     return next();
   } catch (err) {
     return next({
-      log: `errorController.saveErrors: ERROR: ${ err }`,
+      log: `errorController.saveErrors: ERROR: ${err}`,
       message: {
         err: 'Error occurred while saving errors. Check server logs for more information.',
-      }
-    })
+      },
+    });
   }
 };
 
 errorsController.getErrors = async (req, res, next) => {
   try {
+    await K8sError.deleteOne({ namespace: 'NAMESPACE' });
```

Comment:
Are we adding the 'NAMESPACE' row to the DB, then deleting it as our solution? Or was this just for removing it from the DB from our old tests. If the former, we should figure out a better way to handle this; if that latter, we  should remove this line.

Reply:
Yeah it was our bandaid solution for MVP but came up with a better solution in the newest commit.